docs/ocaml/00-outline.md

```diff
@@ -0,0 +1,143 @@
+1. Intro
+  - Why Functional Programming?
+    - mutability
```

Comment:
immutability ?

Reply:
Refers to talking about why mutability is bad. See the introduction section for a more fleshed out explanation.